drivers/clock_control/clock_stm32f0_f3.c

```diff
@@ -150,6 +150,9 @@ void config_enable_default_clocks(void)
 	LL_APB1_GRP2_EnableClock(LL_APB1_GRP2_PERIPH_SYSCFG);
 #endif
 #else
+	/* Enable PWR clock, required to access BDCR */
+	LL_APB1_GRP1_EnableClock(LL_APB1_GRP1_PERIPH_PWR);
+
```

Comment:
I simply moved it to beginning of function to get it works with stm32f072 too.

Reply:
Thanks @katyo. Would you mind contributing your fix ?